src/Dropdown/Dropdown.tsx

```diff
@@ -29,6 +29,16 @@ type DefaultProps = {
   placeholder?: string
   /** label displayed above the dropdown  */
   label?: string
+  /** label typography displayed above input  */
+  labelTypo?: string
```

Comment:
Just to sums up what I think we should do here is to just keep this `labelTypo`, decide with design which other typo of the one we already have to use for bidder labels and remove all other props to keep it as basic as possible.